lib/HLSL/HLLowerUDT.cpp

```diff
@@ -255,7 +255,7 @@ void hlsl::ReplaceUsesForLoweredUDT(Value *V, Value *NewV) {
       IRBuilder<> Builder(GEP);
       SmallVector<Value *, 4> idxList(GEP->idx_begin(), GEP->idx_end());
       Value *NewGEP = Builder.CreateGEP(NewV, idxList);
-      ReplaceUsesForLoweredUDT(GEP, NewGEP);
+      ReplaceUsesForLoweredUDTImpl(GEP, NewGEP);
       dxilutil::MergeGepUse(NewGEP);
```

Comment:
```suggestion
```

I think we need to get rid of this call if we can.  If it's required for some reason, it would be interesting to understand why.